src/jsonschema/compile_evaluate.cc

```diff
@@ -17,9 +17,18 @@
 
 namespace {
 
+#define PUSH_BACK(weak_pointer, target)                                        \
```

Comment:
```suggestion
#define PUSH_BACK(pointer, target)                                        \
```

The first argument is the non-weak one. In any case, hopefully we move this to `.push_back()` to get rid of the macro